pkgs/applications/networking/mailreaders/mblaze/default.nix

```diff
@@ -1,11 +1,13 @@
-{ stdenv, lib, fetchFromGitHub, fetchpatch, libiconv, ruby ? null }:
+{ stdenv, lib, fetchFromGitHub, fetchpatch, libiconv, ruby ? null, makeWrapper, coreutils, file, gawk, gnugrep, gnused }:
```

Comment:
Maybe break this long line at 80 or something? And optional arguments (`ruby`) usually come last.